handlers.go

```diff
@@ -283,9 +298,18 @@ func (s *Server) ServeAPI(srv string, logging bool, lsteptok string, cachedir st
 	shuttle.POST("/init", s.handleShuttleInit)
 	shuttle.GET("/list", s.handleShuttleList)
 
+	autoretrieve := admin.Group("/autoretrieve")
+	autoretrieve.POST("/init", s.handleAutoretrieveInit)
+	autoretrieve.GET("/list", s.handleAutoretrieveList)
+
+	e.POST("/autoretrieve/heartbeat", s.handleAutoretrieveHeartbeat)
```

Comment:
given you created an autoretrieve group above, why not use it

Reply:
Also, why have this implementation in this PR? I think focusing on just introducing swagger in this PR would be a better approach and then have another PR for this, so we can properly contextualize PRs.

Reply:
Same as above